src/structures/MessageAttachment.js

```diff
@@ -43,6 +43,22 @@ class MessageAttachment {
     return this;
   }
 
+  /**
+   * Sets or unsets the attachment as a spoiler
+   * @param {boolean} spoiler Whether the attachment should be in a spoiler
```

Comment:
```suggestion
   * Sets whether this attachment is a spoiler
   * @param {boolean} spoiler Whether the attachment should be marked as a spoiler
```
Wording here could be improved to be consistent with other descriptions (e.g., [`.setNSFW`](https://discord.js.org/#/docs/main/stable/class/BaseGuildTextChannel?scrollTo=setNSFW))